A/abPOA/bundled/patches/no_march_and_shared_libs_on.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9e4b642..a324fdb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -10,10 +10,8 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
+ 
+ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+ 
+-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=native") # SIMD
+-
+ # build abPOA as a static library by default
+-set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build all libraries as shared")
++set(BUILD_SHARED_LIBS ON CACHE BOOL "Build all libraries as shared")
```

Comment:
Can't you set this option on the command line instead of changing it in the code?

Reply:
Yeah unfortunately this build is causing me a lot of issues (as evidenced with all that red below :/) so somewhere I tried patching to see if something would work. I'll come back to this pr over the week.